### ISSUE TITLE
Perf Dashboard: Later bisects will not fix the assigned issue on Chromeperf

### ISSUE THREAD

**Issue by natorion** [catapult-project/catapult#3932]
Repro/Workflow:

1.) Create a new issue (https://bugs.chromium.org/p/chromium/issues/detail?id=769693) and start multiple bisects on a large alert page https://chromeperf.appspot.com/group_report?sid=bd95b960d11403ebdb2787317048c729afbe6916e0b32620ef660ac93353cb18 

2.) First bisect results identifies a duplicate (https://bugs.chromium.org/p/chromium/issues/detail?id=769693#c5) and the assigned issue is correct in https://chromeperf.appspot.com/group_report?sid=bd95b960d11403ebdb2787317048c729afbe6916e0b32620ef660ac93353cb18 to 764614

3.) Later bisect results find other duplicates https://bugs.chromium.org/p/chromium/issues/detail?id=769693#c6 and https://bugs.chromium.org/p/chromium/issues/detail?id=769693#c7. The issue on Monorail is corrected but not the assigned issue on Chromeperf in https://chromeperf.appspot.com/group_report?sid=bd95b960d11403ebdb2787317048c729afbe6916e0b32620ef660ac93353cb18. They still point to 764614.

4.) Data on Chromeperf and Monorail is inconsistent at that point.

Potential solutions:

1.) Keep all the alerts in sync with the Monorail issue. If some alert has not the old duplicate assigned (e.g. 764614), don't overwrite it to prevent overwriting manual re-assignements.

2.) Only change the issue assignment of the test bisected to the new one. I like this one better because it helps sorting out multiple regressions on the same range (as in the example).

**Comment by anniesullie** [comment]
@dave-2 @simonhatch @benhenry 

We should probably write up a short doc that covers how bisect/pinpoint ideally would
* Assign bugs
* Cc people
* Mark bugs duplicate
* Deal with multiple culprits found in the same bug.

**Comment by dave-2** [comment]
In the future, when we automatically kick off a bisect on every alert, we will be able to narrow down the alert to its specific commits. Then the bugs are associated with the commits, not the alerts or the bisect jobs.

The mapping will be: **1 alert → 1 bisect → N commits → N bugs**

If an alert or bisect job represents multiple regressions, it's then possible for an alert or a bisect job to have multiple bugs. A bug can be associated with multiple alerts/bisects.

Each bug will have only one assignee (the commit author) and there will be no CCs or duplicate bugs.

**Comment by anniesullie** [comment]
I agree the long-term vision is great, but I think we also need a clear story until we get there. What should the dashboard do when a perf sheriff creates a bug with multiple root causes in the short term?

Long-term, what happens when pinpoint misidentifies a culprit CL? I can imagine some scenario where:
* A bug causes pinpoint to mass-misidentify culprits. For example, maybe there is a misconfiguration in binaries for build-per-revision.
* We need to re-run and re-triage after updating the binaries
